lm75bd/lm75bd.c

```diff
@@ -27,7 +29,30 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
-  
+  if (temp == NULL){
+    return ERR_CODE_INVALID_ARG; //pointer should not be null
+  }
+
+  uint8_t tempPointer = 0x0; //temp pointer reg is set to 0 to specify temperature reading to the device
+
+  error_code_t res = i2cSendTo(devAddr, &tempPointer, sizeof(tempPointer)); // write pointer byte 0x0 to the device address for temperature reading
+
+  if (res != ERR_CODE_SUCCESS){
+    RETURN_IF_ERROR_CODE(res);
+  }
```

Comment:
This if statement is redundant if you declare an errCode variable above the i2c function call and wrap it inside of the LOG_IF_ERROR_CODE macro